src/components/button-more/button-more.js

```diff
@@ -1,6 +1,12 @@
 import template from "./button-more.html";
 import { MDCMenu } from "@material/menu";
 
+import { TagListService } from "../../services/tag-service";
+import { TransactionListService } from "../../services/transaction-service";
+
+import { TagDialogComponent } from "../tag-dialog/tag-dialog";
+import { ConfirmDialogComponent } from "../confirm-dialog/confirm-dialog";
```

Comment:
do you really need it in button?
![image](https://user-images.githubusercontent.com/14272396/37972178-65f7146a-31e0-11e8-833d-f518285c5867.png)